builder/bdwgc.go

```diff
@@ -59,7 +65,6 @@ var BoehmGC = Library{
 			"blacklst.c",
 			"dbg_mlc.c",
 			"dyn_load.c",
-			"finalize.c",
```

Comment:
I think there's no need to exclude finalize.c as it should compile to zero size if GC_NO_FINALIZATION is defined.

Reply:
Correct, but there's no reason to include it either :shrug: